Documentation/Installation.rst

```diff
@@ -0,0 +1,12 @@
+Installation 
+===========
+
+
+Katydid source can be found on the [Project 8 Github repository](https://github.com/project8/katydid).  
```

Comment:
This format for linking doesn't work in restructuredText

Reply:
Whoooppppps. Google and a quick fix in a117873